akka-bbb-apps/src/main/scala/org/bigbluebutton/core/apps/users/UpdateWebcamsOnlyForModeratorCmdMsgHdlr.scala

```diff
@@ -3,22 +3,30 @@ package org.bigbluebutton.core.apps.users
 import org.bigbluebutton.common2.msgs._
 import org.bigbluebutton.core.running.{ LiveMeeting, OutMsgRouter }
 import org.bigbluebutton.core2.MeetingStatus2x
+import org.bigbluebutton.core2.Permissions
```

Comment:
Hi @jfsiebel

Overall I think this looks good but I could not tell where this import is used.

Reply:
It was not used, forgot to remove it. :smiley: